apps/super-app/src/services/SupportServices.ts

```diff
@@ -33,6 +41,21 @@ export interface FacebookLink {
   text: string;
 }
 
+export function getServiceCardAbouts(): ServiceCardAbout[] {
```

Comment:
I think this should go into another file. This is becoming a catch-all for all info coming from Strapi, but this file was intended to be specific to the support detail page. There's going to be some work required to split up the support detail bits from the TWC2 bits, and adding more things will make that process even more complicated.

Reply:
Gotcha. I think we should reuse the root interface and extend it to both SupportDetail and ServiceCard interfaces, what do you think?

Reply:
Discussed on Slack, we agreed to move the 'common' things to a separate object (not a root interface).